frontend/src/components/forgot-password/forgot-password.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+
+import { ForgotPasswordRequestData } from '@autoline/shared/common/types/types';
+import { ButtonFill } from '@components/common/button-fill/button-fill';
+import { InputField } from '@components/common/input-field/input-field';
+import { useAppForm } from '@hooks/hooks';
+import { Alert } from '@mui/material';
+import { useForgotPasswordMutation } from '@store/queries/auth';
+
+import { forgotPassword as forgotPasswordValidationSchema } from '../../../../shared/src/validation-schemas/validation-schemas';
```

Comment:
`@autoline/shared`

Reply:
it does not work for me. dont know why

Reply:
The issue was in the `shared`'s `module` type. The JS files were compiled as `commonjs` modules instead of the `es6` modules - https://github.com/BinaryStudioAcademy/bsa-2022-autoline/pull/41
Please merge the `develop` into this branch and try to use `@autoline/shared` again